modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp

```diff
@@ -56,16 +56,33 @@ if (editKBArticleDisplayContext.isPortletTitleBasedNavigation()) {
 							type="submit"
 						/>
 
-						<clay:button
-							cssClass="mr-3"
-							disabled="<%= editKBArticleDisplayContext.isPending() %>"
-							displayType="primary"
-							id='<%= liferayPortletResponse.getNamespace() + "publishButton" %>'
-							label="<%= editKBArticleDisplayContext.getPublishButtonLabel() %>"
-							name="publishButton"
-							small="<%= true %>"
-							type="submit"
-						/>
+						<c:choose
+						>
+							<c:when test='<%= FeatureFlagManagerUtil.isEnabled("LPS-192286") %>'>
```

Comment:
Hi @sofiamendozalr 

We usually use for the FF the LPS of the epic or the Main Story.
On this case I think that LPS-188060 is more accurate

Reply:
that is true, FF should point to the epic number